plugins/inputs/couchbase/couchbase.go

```diff
@@ -354,6 +354,14 @@ func (cb *Couchbase) addBucketField(fields map[string]interface{}, fieldKey stri
 	fields[fieldKey] = value
 }
 
+func (cb *Couchbase) addBucketFieldChecked(fields map[string]interface{}, fieldKey string, values []float64, index int) {
```

Comment:
Since it's the array that would be `nil`, not the value within the array, I think this is the only way to prevent the panic. And I couldn't modify `addBucketField` since it's used elsewhere for non-array values.

Reply:
Makes sense to me.

Reply:
Oops, I didn't read the code closely enough. Makes sense to me too.